assembl/static2/css/components/timeline.scss

```diff
@@ -66,32 +73,45 @@
       }
     }
   }
+  .timeline-graph {
+    position: relative;
+
+    .timeline-bars {
+      display: inline-block;
+      width: calc(100% - 10px);
+      height: auto;
+      position: absolute;
+      top: 9px;
+      left: 12px;
+
+      .timeline-bar-1 {
```

Comment:
Sorry mate, the name `timeline-bar-1` is not very clear, without having to dig deep into the DOM code. Readability is important. Which part of the timeline does this represent, top/bottom, left/right, etc?